vncviewer/OptionsDialog.cxx

```diff
@@ -902,6 +975,28 @@ void OptionsDialog::handleClipboard(Fl_Widget *widget, void *data)
 #endif
 }
 
+void OptionsDialog::handleFullScreen(Fl_Widget *widget, void *data)
+{
+  OptionsDialog *dialog = (OptionsDialog*)data;
+
+  if (dialog->fullScreenCheckbox->value()) {
+    dialog->fullScreenModeGroup->activate();
+  } else {
+    dialog->fullScreenModeGroup->deactivate();
+  }
```

Comment:
Hmm... so we didn't do this before? Perhaps we shouldn't do it now either. If someone wants to configure things in preparation for toggling fullscreen later?